[RHIDP-7712]: Add configuration files reference to the Configuring title (#2257)
redhat-developer/red-hat-developers-documentation-rhdh: modules/configure_configuring-rhdh/con-configuration-precedence.adoc

:_mod-docs-content-type: CONCEPT

[id="configuration-precedence_{context}"]
= Configuration precedence

> [role="_abstract"]
> When multiple configuration sources are available, {product-short} merges them following specific precedence rules. Understanding these rules helps you predict how your configuration files interact.
> 
> Application configuration layering::
> When you provide multiple `{my-app-config-file}` files (for example, through multiple config maps), {product-short} deep-merges them. Later files override earlier files for the same keys. For object values, the merge is recursive. For array values, the later array replaces the earlier array entirely.

Reviewer: ```suggestion

Review these configuration precedence behaviors:

Application configuration layering::
When you provide multiple `{my-app-config-file}` files (for example, through multiple config maps), {product-short} deep-merges them. Later files override earlier files for the same keys. For object values, the merge is recursive. For array values, the later array replaces the earlier array entirely.
```
(I think having a sentence to introduce the definition list. Or... in line 7, we could say "Understanding the following rules"?)
